src/braket/devices/local_simulator.py

```diff
@@ -39,9 +38,7 @@
 from braket.tasks.local_quantum_task import LocalQuantumTask
 from braket.tasks.local_quantum_task_batch import LocalQuantumTaskBatch
 
-_simulator_devices = {
```

Comment:
Hi, thank you for the contribution!

Seems that Python 3.9 has a [different behavior](https://github.com/python/importlib_metadata/blob/6673723baa487e447135815e8aaf1bb41aa963bd/importlib_metadata/__init__.py#L283) for using `entry_points`

Reply:
@math411 thanks for pointing this out. Updated to use a backport for Python 3.9 as recommend by the offical [doc](https://docs.python.org/3/library/importlib.metadata.html#entry-points) (see the "Compatibility Note" part).